src/odemis/gui/cont/tabs.py

```diff
@@ -628,6 +641,13 @@ def Show(self, show=True):
         assert (show != self.IsShown())  # we assume it's only called when changed
         super(LocalizationTab, self).Show(show)
 
+        # Enable the tab if the stage is in imaging position, disable it otherwise
+        stage = self.tab_data_model.main.stage
+        if getCurrentPositionLabel(stage.position.value, stage) is IMAGING:
+            self.panel.Enable()
+        else:
+            self.panel.Disable()
+
```

Comment:
This whole part is not needed. Show is called when the user has pressed on the tab button (or pressed another tab). It's already too late. We just want the tab button to be disabled.